pkg/util/system/cpu_manager_linux.go

```diff
@@ -0,0 +1,54 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/spf13/pflag"
+	cliflag "k8s.io/component-base/cli/flag"
+)
+
+// return cpu policy, cpu state file path and cpu manager opt
+func GuessCPUManagerOptFromKubelet() (string, string, map[string]string, error) {
+	pids, err := PidOf(Conf.ProcRootDir, "kubelet")
+	if err != nil || len(pids) == 0 {
+		return "", "", nil, fmt.Errorf("failed to find kubelet's pid, kubelet may stop: %v", err)
+	}
+	kubeletPid := pids[0]
+
+	kubeletArgs, err := ProcCmdLine(Conf.ProcRootDir, kubeletPid)
```

Comment:
I remember that kubelet also supports configuring CPU management policies via configuration file.  FYI: https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/

Reply:
add TODO